### PR TITLE
Add official documentation references to tutorials

### PR DESCRIPTION
**Pull request by matthewrsj**
Many users are stumbling on this repository from a google search and
expecting it to

1) be a complete usage documentation on how to develop a Clear Linux
   derivative and
2) that the methods described in this documentation are exhaustive and
   up-to-date.

Add references to direct users to the officially maintained and
centrally managed documentation we expect them to use for most use
cases.

Signed-off-by: Matthew Johnson <matthew.johnson@intel.com>

### CODE REVIEW

File: 04-mixing.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 How To Clear - Mixing content and creating updates
 ==================================================
 
+For official documentation on these topics please see
+[https://clearlinux.org/documentation](clearlinux.org/documentation).
```
Comment (tmarcu) [inline 04-mixing.md]: Should be a link to https://clearlinux.org/documentation/clear-linux/guides/maintenance/mixer directly. Nothing should be shown in this page about how to actually use mixer and commands to run.

Comment (matthewrsj) [reply 04-mixing.md]: done

**Comment by matthewrsj** [comment]
@ahkok ping

**Comment by ahkok** [comment]
8d2fccb3f8863e50da8f82c57991e12781c64317

**Pull request closed without merging**